sdk/core/core-sse/test/internal/createStream.spec.ts

```diff
@@ -32,12 +36,13 @@ describe("createStream", () => {
     assert.isTrue(canceled);
   });
 
-  it("creates disposable stream", async function (this: Context) {
+  it("creates disposable stream", async function () {
     let disposed = false;
     {
-      await using stream = createStream(createIter(), async () => {
```

Comment:
why did we have to move away from this syntax?

Reply:
`vitest` currently does not support the `using` statement, nor does the browser support it yet.